music-blocks.html

```diff
@@ -144,6 +144,34 @@ <h2>CREDITS</h2>
     </div>
 </section>
 
+<!-- "Refence Video" section -->
+<section id="testimonial" >
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12" >
+                <div class="row justify-content-center">
+                    <div class="col-md-8 col-md-push-2">
+                        <h2 class="text-center">REFERENCE VIDEOS</h2>
+                        <p class="customParagraphStyle">Explore more about Sugar labs and Music Blocks on our official youtube channel<a href="https://www.youtube.com/@SugarlabsOrg-EN" class="hrefCustomColor"> Sugar Labs</a>.</p>
```

Comment:
I think we should link to our Music Blocks playlist instead: https://www.youtube.com/watch?v=v8-M81pp5ng&list=PLyTz5XRZyi-xRj6aca64IrXtjLeMydWrW&pp=gAQBiAQB

Reply:
So should I add more such type of iframe or just add a simple clickable link there below